requirements/testing.txt

```diff
@@ -2,9 +2,9 @@
 -r local-docs-build.txt
 
 django-dynamic-fixture==3.1.0
-pytest==5.2.2
+pytest==5.4.2
 pytest-custom-exit-code==0.3.0
-pytest-django==3.6.0
+pytest-django==3.8.0
```

Comment:
I upgraded these because then it gave me proper failures on the migrations instead of blowing up :)